src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/WebServicesCmdletBase.cs

```diff
@@ -138,6 +140,7 @@ protected override void StopProcessing()
             }
             catch (Exception ex)
             {
+                this.WriteVersionInfoToDebugChannel();
```

Comment:
I was thinking more of adding this information to the error itself, but you could do this in a subsequent PR